app/models/integer_constraint.rb

```diff
@@ -0,0 +1 @@
+class IntegerConstraint < Constraint; end
```

Comment:
since STI is used, I wonder if any method could be moved to the subclasses. Maybe you intended to leave this option open for other subclasses. But the two subclasses here don't have any behavior defined in them.

Reply:
This should have been merged into develop rather than master (since updated). As it was, there were too many commits to review. Apologies.

In this case, though, those will get built out over time as we need to handle user input on fields when building grants.